terminator/tokenization.py

```diff
@@ -166,6 +167,32 @@ def tokenize(self, text: str) -> List[str]:
         return self.tokenizer.tokenize(text)
 
 
+class PolymerTokenizer:
```

Comment:
May I suggest to call this in a different way? Probably `PolymerGraphTokenizer` is more appropriate in this case.